pxtblocks/loader.ts

```diff
@@ -108,7 +108,7 @@ export function injectBlocks(blockInfo: pxtc.BlocksInfo): pxtc.SymbolInfo[] {
     return blockInfo.blocks
         .map(fn => {
             const comp = pxt.blocks.compileInfo(fn);
-            const block = createToolboxBlock(blockInfo, fn, comp);
+            const block = createToolboxBlock(blockInfo, fn, comp, false, 2);
```

Comment:
Why 2 here?

Reply:
this is the number of times it's allowed to recurse. I don't think we have any blocks that go more than two shadow blocks deep, so this should cover it